processor/src/test/java/org/mapstruct/ap/test/bugs/_3852/Issue3852Mapper.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.bugs._3852;
```

Comment:
This is not a bug. It's an enhancement. Perhaps you can just add to the tests in `org.mapstruct.ap.test.nestedproperties.simple` and / or put them in some other package

Reply:
done